src/GitHubExtension/Widgets/GitHubIssuesWidget.cs

```diff
@@ -206,9 +213,22 @@ private void DataManagerUpdateHandler(object? source, DataManagerUpdateEventArgs
             if (fullName == e.Description && e.Context.Contains("Issues"))
             {
                 Log.Logger()?.ReportInfo(Name, ShortId, $"Received matching repository update event.");
+
+                // Don't update if we're in configuration mode.

```

Comment:
Can this just move up so we aren't doing the extra string checks?

Reply:
Same with others

Reply:
Done